changelog/14232.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
```

Comment:
I'm catching this in final changelog review, but we've been putting these under the "changes" section, and not the "improvements" section.

Reply:
You also don't need a separate one for ENT, I think?